layers/vulkan/generated/chassis.cpp

```diff
@@ -16378,25 +16386,23 @@ VKAPI_ATTR VkResult VKAPI_CALL QueueSubmit2KHR(VkQueue queue, uint32_t submitCou
     RecordObject record_obj(vvl::Func::vkQueueSubmit2KHR);
     {
         VVL_ZoneScopedN("PreCallRecord_vkQueueSubmit2KHR");
+        VVL_TracyVkNamedZoneStart(GetTracyVkCtx(), queue, "gpu_PreCallRecordQueueSubmit", pre_call_record_gpu_zone);
```

Comment:
if we are going to leave these in, at least match the name correctly

```suggestion
        VVL_TracyVkNamedZoneStart(GetTracyVkCtx(), queue, "gpu_PreCallRecordQueueSubmit2KHR", pre_call_record_gpu_zone);
```